staging/src/k8s.io/legacy-cloud-providers/azure/azure_standard.go

```diff
@@ -316,10 +325,17 @@ func (az *Cloud) serviceOwnsFrontendIP(fip network.FrontendIPConfiguration, serv
 	return strings.HasPrefix(*fip.Name, baseName)
 }
 
-func (az *Cloud) getFrontendIPConfigName(service *v1.Service, subnetName *string) string {
+func (az *Cloud) getFrontendIPConfigName(service *v1.Service) string {
 	baseName := az.GetLoadBalancerName(context.TODO(), "", service)
+	subnetName := subnet(service)
 	if subnetName != nil {
-		return fmt.Sprintf("%s-%s", baseName, *subnetName)
+		ipcName := fmt.Sprintf("%s-%s", baseName, *subnetName)
+		
+		// Azure lb front end configuration name must not exceed 80 charactors
+		if len(ipcName) > 80 {
```

Comment:
there are two places referencing num 80, pls define a const

Reply:
defined 2 constants for rule name max length and ip config name max length.